packages/@stimulus/core/src/action_descriptor.ts

```diff
@@ -7,7 +7,7 @@ export interface ActionDescriptor {
 }
 
 // capture nos.:            12   23 4               43   1 5   56 7      768 9  98
-const descriptorPattern = /^((.+?)(@(window|document))?->)?(.+?)(#([^:]+?))(:(.+))?$/
+const descriptorPattern = /^((.+?)(@(window|document))?(->|&gt;))?(.+?)(#([^:]+?))(:(.+))?$/
```

Comment:
I couldn't regexp myself out of a wet paper bag, and I wonder if this style disrupts the `// capture nos.:` above.